src/main/java/de/tum/in/www1/artemis/domain/GradingInstruction.java

```diff
@@ -0,0 +1,155 @@
+package de.tum.in.www1.artemis.domain;
+
+import java.io.Serializable;
+import java.util.Objects;
+
+import javax.persistence.*;
+
+import org.hibernate.annotations.Cache;
+import org.hibernate.annotations.CacheConcurrencyStrategy;
+
+/**
+ * A Structured Grading Instruction.
+ */
+@Entity
+@Table(name = "grading_instruction")
+@Cache(usage = CacheConcurrencyStrategy.NONSTRICT_READ_WRITE)
+public class GradingInstruction implements Serializable {
```

Comment:
You could add some JavaDoc to the attributes with a short explanation (e.g. the one you are using in your proposal). Having this is always really nice for new developers.